app/renderer/js/components/webview.js

```diff
@@ -55,9 +64,17 @@ class WebView extends BaseComponent {
 				url === domainPrefix + '/' ||
 				url.includes(domainPrefix + '/user_uploads/'); // URL containing the file attachments
 
+			const isImageURL = /\.(png|gif|bmp|webp|jpeg|jpg)/;
```

Comment:
I think we should use the function here - `LinkUtil.isImage(url)`. I'll merge the other PR first so that you just need to rebase and use this function.

Reply:
Yeah, makes sense.